modules/EmailTemplates/Save.php

```diff
@@ -53,7 +53,7 @@
 
 require_once('modules/EmailTemplates/EmailTemplateFormBase.php');
 $form = new EmailTemplateFormBase();
-sugar_cache_clear('select_array:'.$focus->object_name.'namebase_module=\''.$focus->base_module.'\'name');
+sugar_cache_clear('select_array:'.$focus->object_name.'namebase_module=\''. (isset($focus->base_module) ? $focus->base_module : null).'\'name');
```

Comment:
It doesn't matter by functional behavior.
In PHP at string concatenation the `(string)null` value cast to string evaluated as an empty string `''` .
Also in PHP the undefined variables and object properties evaluated as `null` and indicate a php notice. 
so if it's `null` that clearly means it is an undefined variable which evaluated as an empty string without php notice.